source/edge-server/configure.txt

```diff
@@ -490,40 +509,63 @@ To cycle the Edge Server API key:
-automatically syncs all data in all of the collections in your linked data 
-source.
+  .. code-block:: console
 
-If you would like to filter the data that syncs to your Edge Server, you have
-two options:
+     edgectl config --query-file="/Users/my.user/edge-server/query.json"
 
-[PLACEHOLDER - query handling is changing so this will need to be updated
-based on the final implementation]
-- Pass a simple string query. For example, ``--query="truepredicate"``.
-- Pass an object containing one or more collections and queries to sync a 
-  subset of your data with Edge Server. This may resemble:
+- Pass a global query to use the same query for all collections:
 
-  .. code-block:: json
+  .. code-block:: console
 
-     {
-        "YOUR-SCHEMA-TITLE": "YOUR-QUERY",
-        "YOUR-SCHEMA-TITLE": "YOUR-QUERY"
-     }
+     edgectl config --global-query="shouldSync == true"
 
-  - ``YOUR-SCHEMA-TITLE``: The ``title`` field of a :ref:`schema <schemas>`, 
-    which typically matches the name of a collection in your database. If 
-    you're using a template app, this is the ``Item`` collection. Your linked
-    Atlas data source may contain many collections, each with a matching 
-    schema. If you do not use the default Edge Server query ``"*"`` that syncs
-    everything, each schema that you want to sync must have its own query.
+Unless you're using a global query, Edge Server queries take the form of 
+``YOUR-SCHEMA-TITLE: YOUR-QUERY``:
 
-  - ``YOUR-QUERY``: The query that determines which documents the Edge 
-    Server can sync with a collection. You can use :ref:`Realm Query Language <rql>` 
-    to define this query, with the same caveats around :ref:`Flexible Sync RQL 
-    Limitations <flexible-sync-rql-limitations>`.
+- ``YOUR-SCHEMA-TITLE``: The ``title`` field of a :ref:`schema <schemas>`, 
+  which typically matches the name of a collection in your database. If 
+  you're using a template app, this is the ``Item`` collection. Your linked
+  Atlas data source may contain many collections, each with a matching 
+  schema. If you do not use the default Edge Server query ``"*"`` or a global
+  query that syncs everything, each schema that you want to sync must have 
+  its own query.
+
+- ``YOUR-QUERY``: The query that determines which documents the Edge 
+  Server can sync with a collection. You can use :ref:`Realm Query Language <rql>` 
+  to define this query, with the same caveats around :ref:`Flexible Sync RQL 
+  Limitations <flexible-sync-rql-limitations>`.
+
+.. note:: Queryable Fields
+
+   Because Edge Servers use Device Sync to sync data with Atlas, the query you
+   use on the Edge Server instance must use a **queryable field**. If you use
+   an Edge Server Service, we automatically add fields that you use in Edge
+   Server instance queries to the queryable fields. If you use a Device Sync
+   service with Edge Servers, we only add queryable fields when Development
```

Comment:
I see you capitalize service in "Edge Server Service" but not for Device Sync service". Is this correct?

Reply:
That is a good question! The capitalization of "S" in service is inconsistent throughout the UI, and I've been similarly inconsistent in the docs. Probably worth an audit at some point. But I'll fix this instance.